lib/fuzzy-finder-view.js

```diff
@@ -292,26 +267,29 @@ export default class FuzzyFinderView {
     }
   }
 
-  setItems (filePaths) {
-    this.items = this.projectRelativePathsForFilePaths(filePaths)
+  setItems (items) {
+    this.items = items
     if (this.isQueryALineJump()) {
       return this.selectListView.update({items: [], loadingMessage: null, loadingBadge: null})
     } else {
       return this.selectListView.update({items: this.items, loadingMessage: null, loadingBadge: null})
     }
   }
 
+  // THEN: Identify any places in fuzzy-finder that break when the item is not a file on disk (i.e., when it doesn't have a valid local path)
```

Comment:
@jasonrudolph @as-cii -- Note to selves: Remove this comment before merging this pull request.